Hazelnut/src/Panels/SceneHierarchyPanel.cpp

```diff
@@ -323,6 +339,41 @@ namespace Hazel {
 		DrawComponent<SpriteRendererComponent>("Sprite Renderer", entity, [](auto& component)
 		{
 			ImGui::ColorEdit4("Color", glm::value_ptr(component.Color));
+
+			// Texture
+			{
+				const uint32_t id = component.Texture == nullptr ? 0 : component.Texture->GetRendererID();
+
+				ImGui::PushStyleVar(ImGuiStyleVar_ItemSpacing, ImVec2{ 0, 0 });
+
+				ImGui::Text("Texture");
+				const ImVec2 buttonSize = { 80, 80 };
+				ImGui::SameLine(ImGui::GetWindowWidth() * 0.6f);
+
+				ImGui::PushStyleColor(ImGuiCol_Button, ImVec4{ 0.25f, 0.25f, 0.25f, 1.0f });
+				ImGui::PushStyleColor(ImGuiCol_ButtonHovered, ImVec4{ 0.35f, 0.35f, 0.35f, 1.0f });
+				ImGui::PushStyleColor(ImGuiCol_ButtonActive, ImVec4{ 0.25f, 0.25f, 0.25f, 1.0f });
+				if(ImGui::ImageButton((ImTextureID)id, buttonSize, { 0, 0 }, { 1, 1}, 0))
```

Comment:
This (line - 356) is wrong. The image is inverted in this method. This should be ⏬ 
`if (ImGui::ImageButton((ImTextureID)id, buttonSize, { 0, 1 }, { 1, 0 }, 0, {1, 0, 1, 1}))`